docs-ref-conceptual/azure-cli-endpoints.md

```diff
@@ -137,29 +137,37 @@ synapse_analytics_endpoint | *.dev.azuresynapse.azure.cn
 
 Azure CLI extensions are optional and installed separately.  The Azure CLI uses https://aka.ms/azure-cli-extension-index-v1 to fetch a list of extensions.  This _aka.ms_ link points to https://github.com/Azure/azure-cli/blob/3feea02888ea67f033f407174a3a7a340158b81a/src/azure-cli-core/azure/cli/core/extension/_index.py#L11.  
 
-All extensions install with endpoint **azcliprod.blob.core.windows.net** and **azurecliextensionsync.blob.core.windows.net** with the following exceptions:
+All extensions install with endpoint **azcliprod.blob.core.windows.net** and **azcliextensionsync.blob.core.windows.net** with the following exceptions:
```

Comment:
`azcliprod` save the wheel file, `azcliextensionsync` save the index.
`azurecliextensionsync` saves the command tree (`https://azurecliextensionsync.blob.core.windows.net/cmd-index/extensionCommandTree.json`), which is used in dynamic installation. I'm unsure if this information should be included in the documentation.

Reply:
I thought about that part too but felt better not including dynamic installation into this page. Because this page talked about configuring extension endpoints proactively in extension installation, while `azurecliextensionsync` is used during extension dynamic installation **passively**.

If we plan to introduce the `azurecliextensionsync` endpoints, I feel we need to explain what dynamic installation is and that would be a different topic? 

I'm very happy to know if anyone has some ideas about this part.